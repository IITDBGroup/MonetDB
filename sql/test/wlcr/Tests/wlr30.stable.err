stderr of test 'wlr30` in directory 'sql/test/wlcr` itself:


# 10:21:36 >  
# 10:21:36 >  "/usr/bin/python2" "wlr30.py" "wlr30"
# 10:21:36 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 51217
# cmdline opt 	mapi_usock = /var/tmp/mtest-27853/.s.monetdb.51217
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/branches/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone
#WLR scheduler forked
#WLR replicator initializing
#WLRgetConfig master=mTests_sql_test_wlcr
#WLRgetConfig batches=7
#WLRgetConfig tag=6
#WLRgetConfig beat=0
#Run the replicator 0 101
#WLRgetConfig master=mTests_sql_test_wlcr
#WLRgetConfig batches=7
#WLRgetConfig tag=6
#WLRgetConfig beat=0
#client1:!ERROR:MALException:sql.replicate:Stop tag limit should be positive or timestamp should be set
#WLRgetConfig master=mTests_sql_test_wlcr
#WLRgetConfig batches=7
#WLRgetConfig tag=6
#WLRgetConfig beat=0
#WLRput: batches 7 tag 6 limit 8 beat 0 timelimit 
#replicate: wait until wlr_limit = 8 (tag 6) time  (2019-10-02T15:01:51.000)
#replicate wait state 101 wlr_limit 8 (wlr_tag 6) wlc_tag 9 wlr_batches 7
#replicate wait state 101 wlr_limit 8 (wlr_tag 6) wlc_tag 9 wlr_batches 7
#Ready to start the replay against batches state 101 wlr 6  wlr_limit 8  wlc 9  taglimit 8
#REPLAY LOG FILE:/export/scratch1/mk/branches/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000007
#do transaction tag 7 wlr_limit 8 wlr_tag 6
#run against tlimit   wlr_tag 6  tag7 
#process a transaction






#WLRput: batches 7 tag 7 limit 8 beat 0 timelimit 
#unexpected instruction =function user.wlr():void;
#wlr.process:processed log file  wlr_tag 7 wlr_limit 8 time 
#WLRput: batches 8 tag 7 limit 8 beat 0 timelimit 
#REPLAY LOG FILE:/export/scratch1/mk/branches/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000008
#do transaction tag 8 wlr_limit 8 wlr_tag 7
#run against tlimit   wlr_tag 7  tag8 
#process a transaction











#WLRput: batches 8 tag 8 limit 8 beat 0 timelimit 
#wlr.process:processed log file  wlr_tag 8 wlr_limit 8 time 
#WLRput: batches 9 tag 8 limit 8 beat 0 timelimit 
#replicate finished 8 (tag 8)
#WLR stop replication
#Replicator thread is stopped 
MAPI  = (monetdb) /var/tmp/mtest-27853/.s.monetdb.51217
QUERY = call wlr.replicate(-1);
ERROR = !Stop tag limit should be positive or timestamp should be set


# 10:21:36 >  
# 10:21:36 >  "Done."
# 10:21:36 >  

