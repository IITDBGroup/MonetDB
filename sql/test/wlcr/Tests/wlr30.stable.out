stdout of test 'wlr30` in directory 'sql/test/wlcr` itself:


# 10:21:36 >  
# 10:21:36 >  "/usr/bin/python2" "wlr30.py" "wlr30"
# 10:21:36 >  

# MonetDB 5 server v11.26.0 (hg id: cbfc9f47e58c)
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr-clone', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:55121/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-29054/.s.monetdb.55121
# MonetDB/GIS module loaded
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000010
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000011
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000012
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000013
# MonetDB/SQL module loaded
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 1,	5 # length
[ 1,	"hello"	]
[ 2,	"world"	]
[ 3,	"blah"	]
[ 4,	"bloh"	]
[ 5,	"red"	]
[ 6,	"fox"	]
#call replicate('mTests_sql_test_wlcr');
#Waiting for replay scheduler to stop
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000004
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000005
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 1,	4 # length
[ 3,	"blah"	]
[ 2,	"blah"	]
[ 3,	"blah"	]
[ 4,	"blah"	]
[ 5,	"blah"	]
[ 6,	"blah"	]
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc.config
#logs=/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/
#id=6
#write=2019-08-14T12:11:24.000
#state=1
#batches=6
#beat=0
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000004
#    wlr.transaction(4:lng, "2019-08-14T12:11:24.000":str, "monetdb":str);
#    wlr.action("update tmp set i = 3 where i = 1;":str);
#    wlr.update("sys":str, "tmp":str, "i":str, 0@0:oid, 3:int);
#    wlr.commit();
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000003
#    wlr.transaction(3:lng, "2019-08-14T12:11:21.000":str, "monetdb":str);
#    wlr.action("insert into tmp values(5,\\'red\\'),(6,\\'fox\\');":str);
#    wlr.append("sys":str, "tmp":str, "i":str, 5:int, 6:int);
#    wlr.append("sys":str, "tmp":str, "s":str, "red":str, "fox":str);
#    wlr.commit();
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000001
#    wlr.transaction(1:lng, "2019-08-14T12:11:18.000":str, "monetdb":str);
#    wlr.action("insert into tmp values(1,\\'hello\\'), (2,\\'world\\');":str);
#    wlr.append("sys":str, "tmp":str, "i":str, 1:int, 2:int);
#    wlr.append("sys":str, "tmp":str, "s":str, "hello":str, "world":str);
#    wlr.commit();
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000005
#    wlr.transaction(5:lng, "2019-08-14T12:11:24.000":str, "monetdb":str);
#    wlr.action("update tmp set s = \\'blah\\';":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 0@0:oid, "blah":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 1@0:oid, "blah":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 2@0:oid, "blah":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 3@0:oid, "blah":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 4@0:oid, "blah":str);
#    wlr.update("sys":str, "tmp":str, "s":str, 5@0:oid, "blah":str);
#    wlr.commit();
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000002
#    wlr.transaction(2:lng, "2019-08-14T12:11:19.000":str, "monetdb":str);
#    wlr.action("insert into tmp values(3,\\'blah\\'),(4,\\'bloh\\');":str);
#    wlr.append("sys":str, "tmp":str, "i":str, 3:int, 4:int);
#    wlr.append("sys":str, "tmp":str, "s":str, "blah":str, "bloh":str);
#    wlr.commit();
#
/export/scratch1/mk/branches/default//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs/mTests_sql_test_wlcr_000000000000
#    wlr.transaction(0:lng, "2019-08-14T12:11:18.000":str, "monetdb":str);
#    wlr.catalog("create table tmp(i int, s string);":str);
#    wlr.commit();
#

# 10:21:36 >  
# 10:21:36 >  "Done."
# 10:21:36 >  

